src/components/Display.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Display = () => <div className="display" />;
+
+export default Display;
```

Comment:
- it should accepts the result of the calculation as an incoming prop [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)]

- Make this `result` value to be string using `propTypes`  ( [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png) )

- make this `result` default value to be `0` using `defaultProps` (see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))

Reply:
Done. Thanks